### PR TITLE
clickable url

### PR DESCRIPTION
**Pull request by Noy-Maimon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc#32330

## Description
making the url in the warroom entry clickable 

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Review by reutshal — APPROVED** [review]
Nice!
Did you verify this in the war room?

**Comment by Noy-Maimon** [comment]
@reutshal yes, verified

**Comment by Noy-Maimon** [comment]
@roysagi  Hey, can you please take a look at the release notes? thanks

File: Packs/PhishTank/ReleaseNotes/2_0_3.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### PhishTank v2
+The 'Additional details' url in the warroom entry is now clickable.
```
Comment (roysagi) [inline Packs/PhishTank/ReleaseNotes/2_0_3.md]: @Noy-Maimon - please provide more detail here - in what command are we talking? Also please use the RN documentation template.

Comment (Noy-Maimon) [reply Packs/PhishTank/ReleaseNotes/2_0_3.md]: @roysagi Thanks, updated now - can you please verify? thanks again

**Pull request merged**